main.js

```diff
@@ -1,60 +1,58 @@
-const quotes = [
-  {text: 'Leave the gun. Take the cannoli.', movie: 'The Godfather'},
-  {text: 'I\’m gonna make him an offer he can\’t refuse.', movie: 'The Godfather'},
-  {text: 'I want all of you to enjoy your cake, so...enjoy.', movie: 'The Godfather'},
-  {text: 'Frankly my dear, I don\’t give a damn.', movie: 'Gone with the Wind'},
-  {text: 'You don\’t understand! I coulda had class. I coulda been a contender. I could\’ve been somebody, instead of a bum, which is what I am.', movie: 'On the Waterfront'},
-  {text: 'Toto, I\’ve got a feeling we\’re not in Kansas anymore.', movie: 'The Wizard of Oz'},
-  {text: 'Here\’s looking at you, kid.', movie: 'Casablanca'},
-  {text: 'As God is my witness, I\’ll never be hungry again.', movie: 'Gone with the Wind'},
-  {text: 'Say \‘hello\’ to my little friend!', movie: 'Scarface'},
-  {text: 'I feel the need — the need for speed!', movie: 'Top Gun'},
-  {text: 'Houston, we have a problem.', movie: 'Apollo 13'},
-  {text: 'Nobody puts Baby in a corner.', movie: 'Dirty Dancing'},
-  {text: 'I\’ll get you, my pretty, and your little dog, too!', movie: 'The Wizard of Oz'},
-  {text: 'One morning I shot an elephant in my pajamas. How he got in my pajamas, I don\’t know.', movie: 'Animal Crackers'},
-  {text: 'You\’ve got to ask yourself one question: \‘Do I feel lucky?\’ Well, do ya, punk?', movie: 'Dirty Harry'},
-  {text: 'Hold on to your butts.', movie: 'Jurassic Park'},
-  {text: 'As far back as I can remember, I always wanted to be a gangster.', movie: 'Goodfellas'},
-  {text: 'Human sacrifice! Dogs and cats living together. Mass hysteria!', movie: 'Ghostbusters'}
-];
 const displayArea = document.getElementById('display-area');
 
-bindEvents();
-displayNewQuote();
+const newQuoteButton = document.getElementById('new-quote-button');
+newQuoteButton.addEventListener('click', handleNewQuoteClick);
 
-function bindEvents() {
-  const newQuoteButton = document.getElementById('new-quote-button');
-  newQuoteButton.addEventListener('click', handleNewQuoteClick);
-}
+displayNewQuote();
 
 function displayNewQuote() {
-  const quotePara = document.getElementById('quote');
-  const refCite = document.getElementById('ref');
-  const quote = getRandomQuote(quotes);
+  fetch('https://movie-quote-api.herokuapp.com/v1/quote?censored')
+    .then(response => {
+      if (!response.ok) {
+        const quoteBox = document.getElementById('quote-box');
+        quoteBox.textContent = `Network request failed with status ${response.status}: ${response.statusText}. Please try again.`;
+      }
+      return response.json();
+    })
+    .then(quoteJson => {
```

Comment:
if you are looking for the next challenge: maybe rewrite this as [async/await](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await). if you do so, maybe let us merge this merge request first and then start a new one. i think having more split MRs makes the evolution of the code clearer.